### PR TITLE
Added absorbed dose of ionizing radiation units

### PR DESCRIPTION
**Pull request by Jagailo**

### CODE REVIEW

File: Common/UnitDefinitions/AbsorbedDoseOfIonizingRadiation.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+          "Culture": "en-US",
+          "Abbreviations": [ "Gy" ]
+        },
+        {
+          "Culture": "ru-RU",
+          "Abbreviations": [ "Гр" ]
+        },
+        {
+          "Culture": "zh-CN",
+          "Abbreviations": [ "戈瑞" ]
```
Comment (angularsen) [inline Common/UnitDefinitions/AbsorbedDoseOfIonizingRadiation.json:34]: Are you sure about the Chinese?  
I have [recently learned](https://github.com/angularsen/UnitsNet/issues/1171) that they typically use international abbreviations, so maybe "Gy" ? I don't know.



I can't verify the Russian one either, but it seems to translate to English "Gr" ?

File: Common/UnitDefinitions/AbsorbedDoseOfIonizingRadiation.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      ]
+    },
+    {
+      "SingularName": "Rad",
+      "PluralName": "Rads",
+      "FromUnitToBaseFunc": "{x} / 100",
+      "FromBaseToUnitFunc": "{x} * 100",
+      "XmlDocSummary": "The rad is a unit of absorbed radiation dose, defined as 1 rad = 0.01 Gy = 0.01 J/kg.",
+      "XmlDocRemarks": "https://en.wikipedia.org/wiki/Rad_(radiation_unit)",
+      "Prefixes": [ "Kilo", "Mega" ],
```
Comment (angularsen) [inline Common/UnitDefinitions/AbsorbedDoseOfIonizingRadiation.json:45]: `Milli` may also be widely used?

Mentioned here:
https://www.translatorscafe.com/unit-converter/en-US/radiation-absorbed-dose/7-1/gray-rad/
https://www.nuclear-power.com/nuclear-engineering/radiation-protection/absorbed-dose/rad-gray-conversion-units/

File: UnitsNet.Tests/CustomCode/AbsorbedDoseOfIonizingRadiationTests.cs [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        protected override double KiloradsInOneGray => 0.1;
+        protected override double MegagraysInOneGray => 1E-06;
+        protected override double MegaradsInOneGray => 1E-4;
+        protected override double MicrograysInOneGray => 1E+6;
+        protected override double MilligraysInOneGray => 1E+3;
+        protected override double NanograysInOneGray => 1E+9;
+        protected override double PetagraysInOneGray => 1E-15;
+        protected override double PicograysInOneGray => 1E+12;
+        protected override double RadsInOneGray => 100;
+        protected override double TeragraysInOneGray => 1E-12;
```
Comment (angularsen) [inline UnitsNet.Tests/CustomCode/AbsorbedDoseOfIonizingRadiationTests.cs:42]: Verified these

**Review by angularsen — CHANGES_REQUESTED** [review]
Looks good to me, just a few suggestions

File: Common/UnitDefinitions/AbsorbedDoseOfIonizingRadiation.json (hunk 1 above)
Comment (Jagailo) [reply Common/UnitDefinitions/AbsorbedDoseOfIonizingRadiation.json:34]: > Are you sure about the Chinese?

Yes, you're right, I clarified that 戈瑞 is used to translate Gray, but the abbreviation is Gy.
I'll remove this localization.

> I can't verify the Russian one either, but it seems to translate to English "Gr" ?

I speak Russian fluently, can confirm this translation ([wiki](https://ru.wikipedia.org/wiki/%D0%93%D1%80%D0%B5%D0%B9_(%D0%B5%D0%B4%D0%B8%D0%BD%D0%B8%D1%86%D0%B0_%D0%B8%D0%B7%D0%BC%D0%B5%D1%80%D0%B5%D0%BD%D0%B8%D1%8F))).
image

**Comment by Jagailo** [comment]
image

For your information, several tests fail, because I have Russian Windows installed and environment culture is used for localization. I could fix this in a separate PR if needed.

File: UnitsNet.Tests/CustomCode/AbsorbedDoseOfIonizingRadiationTests.cs (hunk 3 above)
Comment (Jagailo) [reply UnitsNet.Tests/CustomCode/AbsorbedDoseOfIonizingRadiationTests.cs:43]: What do you mean? Write some additional tests?

Comment (angularsen) [reply UnitsNet.Tests/CustomCode/AbsorbedDoseOfIonizingRadiationTests.cs:43]: No just a note to myself and any other reviewers, that I have verified these constants 👍

**Comment by angularsen** [comment]
> image
> For your information, several tests fail, because I have Russian Windows installed and environment culture is used for localization. I could fix this in a separate PR if needed.

Interesting, yes please do take a look at fixing those tests in a separate pull request, that would be great!

File: UnitsNet.Tests/CustomCode/AbsorbedDoseOfIonizingRadiation.json [hunk 4]
```diff
@@ -0,0 +1,54 @@
+{
```
Comment (angularsen) [inline UnitsNet.Tests/CustomCode/AbsorbedDoseOfIonizingRadiation.json:1]: It seems you mistakenly copied this file to UnitsNet.Tests?

Comment (Jagailo) [reply UnitsNet.Tests/CustomCode/AbsorbedDoseOfIonizingRadiation.json]: True, didn't notice that

**Comment by angularsen** [comment]
Thanks, great work on this!

Nuget should be out shortly.
[Release UnitsNet/5.18.0 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F5.18.0)

**Pull request merged**